homeassistant/components/reolink/host.py

```diff
@@ -416,6 +421,11 @@ async def update_states(self) -> None:
             wake = True
             self.last_wake = time()
 
+        if self._api.baichuan.privacy_mode():
```

Comment:
Just so I understand: This is here to essentially re-check if privacy mode is still enabled? ( since the local `privacy_mode` value may be stale)?

Reply:
Yes, although on the model I tested the privacy mode status is pushed, I am not 100% sure all models will push this state.
Therefore this is the polling each 1 minute to check the status as a fallback.

The local variable inside HA is only used to keep track if there was a change of the privacy mode or not. (To fire a full state update if the privacy mode changes from on to off). The actual state is stored in the upstream library.